documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_HTTP-REST-API.adoc

```diff
@@ -279,7 +281,12 @@ h|Operation Name h|HTTP Verb h|REST-Path h|Comment (e.g. optional query paramete
 |GetAssetAdministrationShell |GET |/aas |content-suffix: $reference
 |PutAssetAdministrationShell |PUT | |
 |GetAllSubmodelReferences |GET |/aas/submodel-refs |Pagination
-|PostSubmodelReference |POST |/aas/submodel-refs |
+|PostSubmodelReference |POST |/aas/submodel-refs a| Location header of the response contains the value ‘<baseUrl>/aas/submodel-ref/{submodelIdentifier}’ 
```

Comment:
```suggestion
|PostSubmodelReference |POST |/aas/submodel-refs a| Location header of the response contains the value ‘<baseUrl>/aas/submodel-refs/{submodelIdentifier}’ 
```

Reply:
I cannot find the original line without the 's' in the branch anymore. Seems fixed to me, therefore, I will not commit the suggestion.